driver.go

```diff
@@ -318,7 +318,9 @@ func isThinlyProvisioned(vgName, lvName string) (bool, error) {
 	}
 	w.Close()
 	if err := cmd2.Wait(); err != nil {
-		return false, err
+		if err.Error() != "exit status 1" {
```

Comment:
checking strings in Golang is generally bad and I'm not sure any grep implementation in distros is returning `exit status 1`, so I'd say, try with:

https://github.com/docker/docker/blob/master/pkg/integration/utils.go#L159

which calls `ProcessExitCode(err error)` here:

https://github.com/docker/docker/blob/master/pkg/integration/utils.go#L35